docs/automations/integrations/jit/review-jit-alerts/README.md

```diff
@@ -0,0 +1,36 @@
+# Review Jit Security Alerts
+Manage review assignment for high and medium risk Jit security alerts.
+
+<div class="automationImage" markdown="1">
+![Label Jit Alerts](/automations/integrations/jit/review-jit-alerts/review-jit-alerts.png)
+</div>
+<div class="automationDescription" markdown="1">
+
+!!! info "Configuration Description"
+        **Review Jit High Alerts**
+
+        * Conditions (all must be true):
+            * Jit reports one or more high vulnerabilities for the PR
+        * Automation Actions:
+            * Require a review from the organization’s security team.
```

Comment:
Shouldn't we want to require 2 reviewers in this case as well?

Reply:
I mostly wanted to illustrate how to implement two separate policies based on the alert level, rather than to make definitive prescriptions about what the correct way to approach each situation is. In this situation, I also felt that requiring a review from the security team is a stronger enforcement than 2 reviews that could come from anyone, with the idea being that the PR author would be incentivized to fix the vulnerability rather than ping the security team.

However, If you feel strongly about the change, I'm happy to make it.

Reply:
My only concern is that users will understand that it checks only for medium issues when requesting two assignees and only High for a reviewer.
Sometimes users may assume that medium issues include high as well